data/quarg/quarg outfits.txt

```diff
@@ -126,11 +129,12 @@ outfit "Medium Graviton Steering"
 	"mass" 50
 	"outfit space" -50
 	"engine capacity" -50
-	"turn" 1600
+	"turn" 1800
 	"turning energy" 10
-	"turning heat" 5
+	"turning heat" 2
+	"force protection" 0.1
```

Comment:
Ditto on the above for force protection/drag reduction

Reply:
Just did add that one
I'm no balance king I first wanted to see how ppl would react to these two new things being used at all, also I dont want the ships to not get pushed away at all once they're disabled

Reply:
Do note that 0.1 means 10%, meaning the two do 0.2 for a total of 16.7% resistance.
Could bring it higher for sure.

Reply:
With both force protection and drag/inertia reduction, I'm okay with the relatively low amount.